core/lib/types/src/commitment.rs

```diff
@@ -218,185 +250,194 @@ impl SerializeCommitment for StateDiffRecord {
     }
 }
 
-/// Block Output produced by Virtual Machine
-#[derive(Debug, Clone)]
-struct L1BatchAuxiliaryOutput {
-    // We use initial fields for debugging
-    #[allow(dead_code)]
-    l2_l1_logs: Vec<UserL2ToL1Log>,
-    #[allow(dead_code)]
-    initial_writes: Vec<InitialStorageWrite>,
-    #[allow(dead_code)]
-    repeated_writes: Vec<RepeatedStorageWrite>,
-
-    l2_l1_logs_compressed: Vec<u8>,
-    l2_l1_logs_linear_hash: H256,
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+struct L1BatchAuxiliaryCommonOutput {
     l2_l1_logs_merkle_root: H256,
-
-    // Once cut over to boojum, these fields are no longer required as their values
-    // are covered by `state_diffs_compressed` and its hash.
-    // Task to remove: PLA-640
-    initial_writes_compressed: Vec<u8>,
-    initial_writes_hash: H256,
-    repeated_writes_compressed: Vec<u8>,
-    repeated_writes_hash: H256,
-
-    // The fields below are necessary for boojum.
-    system_logs_compressed: Vec<u8>,
-    system_logs_linear_hash: H256,
-    state_diffs_hash: H256,
-    state_diffs_compressed: Vec<u8>,
-    bootloader_heap_hash: H256,
-    events_state_queue_hash: H256,
     protocol_version: ProtocolVersionId,
 }
 
+/// Block Output produced by Virtual Machine
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
```

Comment:
Are `Serialize` / `Deserialize` derives here and in other types in this module only required for tests? If so, I'd suggest making this clear:

```rust
#[derive(Debug, Clone, Eq, PartialEq)]
#[cfg_attr(test, derive(Serialize, Deserialize))]
enum L1BatchAuxiliaryOutput {
     // ...
}
```

...so that the serialization format doesn't get relied upon somewhere eventually.